src/index.html

```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main>
+      <div class="container">
+        <div class="container__calendar calendar calendar--start-day--fri calendar--month-length--30">
```

Comment:
Logically if it has such a container__calendar name, then it should be another element which will be the parent container for the calendar itself. Please remove this class and move the corresponding rule to calendar class.